scripts/deploy/src/install.js

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs';
+import child_process from 'child_process';
+import fetch from 'node-fetch';
+import yaml from 'js-yaml';
+import path from 'path';
+
+const MEDIC_REPO_NAME = 'medic';
+const MEDIC_REPO_URL = 'https://docs.communityhealthtoolkit.org/helm-charts';
+const CHT_CHART_NAME = `${MEDIC_REPO_NAME}/cht-chart-4x`;
+const DEFAULT_CHART_VERSION = '1.0.*';
+
+import { fileURLToPath } from 'url';
+import { obtainCertificateAndKey, createSecret } from './certificate.js';
+import UserRuntimeError from './error.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const readFile = function(f) {
```

Comment:
I'm not sure if it's worth defending against, but I passed in a project_name and namespace of this:

```
project_name: mrjones-delete
namespace: "mrjones-delete"
...
   host: "mrjones-delete.dev.medicmobile.org"
```

And it created it OK apparently:

```
➜  deploy git:(cht-deploy-fixes) ✗ ./cht-deploy -f  ~/Documents/MedicMobile/medic-infrastructure/terraform/aws/dev/cht-projects/mrjones-dev-cht-deploy-values.yaml
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "medic" chart repository
Update Complete. ⎈Happy Helming!⎈
Release does not exist. Performing install.
NAME: mrjones-delete
LAST DEPLOYED: Tue May 28 15:28:12 2024
NAMESPACE: mrjones-delete
STATUS: deployed
REVISION: 1
TEST SUITE: None
Instance at mrjones-delete.dev.medicmobile.org installed successfully.
```

But I can't get to it at `https://mrjones-delete.dev.medicmobile.org/` - it just 404s. I think this is something endemic to our EKS setup and most other folks won't benefit from improvements here  - but wanted to mention it in case it was helpful!

Switching to sane values it worked as expected \o/

```
project_name: mrjones-dev
namespace: "mrjones-dev"
...
   host: "mrjones.dev.medicmobile.org"
```